### ISSUE TITLE
Projects in other repos

### ISSUE THREAD

**Issue by trytomakeyouprivate** [iaacornus/Fedora-OSTree-Setup#94]
- [Distrobox autosetup](https://github.com/trytomakeyouprivate/Distrobox-autosetup)
- [easy bash loops](https://github.com/trytomakeyouprivate/easy-bash-loops)
- [Polkit helper](https://github.com/trytomakeyouprivate/Polkit-helper)
- [Arkenfox setup](https://github.com/trytomakeyouprivate/Arkenfox-softening)
- [Flatpak remote list](https://github.com/trytomakeyouprivate/Flatpak-remotes)
- [COPR command](https://github.com/trytomakeyouprivate/COPR-command) ("the command needs to be integrated to rpm-ostree" the fedora people say, so we need this workaround for the next years lol)
- [Simple Firefox menus](https://github.com/trytomakeyouprivate/Simple-Firefox-Menus)

Repos of other nice people

- [Thunderbird user.js](https://github.com/trytomakeyouprivate/thunderbird-user.js) (mostly, some softening needs to be applied as its too restrictive!)
- [KDE Template manager](https://github.com/trytomakeyouprivate/KDE-5-Template-Manager)
- [bash / fish aliasses](https://github.com/trytomakeyouprivate/bash_aliases)
- [silverblue updater](https://github.com/trytomakeyouprivate/silverblue-update)
- [sddm2rpm](https://github.com/trytomakeyouprivate/sddm2rpm)
- [grub2 themes](https://github.com/trytomakeyouprivate/grub2-themes)

**Comment by john-ghatas** [comment]
Maybe an idea to incorporate uBlue images? https://ublue.it/

**Comment by trytomakeyouprivate** [comment]
Thats nice! never heard of it! This would extend the usability of the OSTree Distros a lot, and if they may have less restrictions even avoid needing to layer codecs and stuff.

**Comment by iaacornus** [comment]
> Thats nice! never heard of it! This would extend the usability of the OSTree Distros a lot, and if they may have less restrictions even avoid needing to layer codecs and stuff.

yep, hence i try to avoid layering as much as possible which is why i persist flatpak first then toolbox. `rpm-ostree` recalculates every dependencies every operation, therefore the more you add and change, the longer time it would take, thereby decreasing its performance.
